### ISSUE TITLE
Fix PDF cors

### ISSUE THREAD

**Issue by kiznick** [kiznick/ratchagitja-editor#1]
Unable to fetch pdf because of cors.
```
Access to fetch at 'https://domain.tld/documents/140D050S0000000006400.pdf' from origin 'http://127.0.0.1:5173' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by kiznick** [comment]
Now I have solution for this issue
Use https://github.com/Rob--W/cors-anywhere/tree/master as middleman request to remote file.
It will increase latency and not safe (if not self-host).
but now I will use it for testing.

**Comment by kiznick** [comment]
Fix in commit: fccd5d8
